app/Models/BeatmapDiscussion.php

```diff
@@ -143,19 +143,25 @@ public function refreshKudosu($event)
             return;
         }
 
-        DB::transaction(function () use ($change, $event, $currentVotes) {
+        DB::transaction(function () use ($change, $event, $eventExtraData, $currentVotes) {
             if ($event === 'vote') {
                 if ($change > 0) {
                     $beatmapsetEventType = BeatmapsetEvent::KUDOSU_GAIN;
                 } else {
                     $beatmapsetEventType = BeatmapsetEvent::KUDOSU_LOST;
                 }
+
+                $eventExtraData['votes'] = $this
+                    ->beatmapDiscussionVotes()
+                    ->get()
```

Comment:
`->beatmapDiscussionVotes`  is already a Collection so `()->get()` seems unnecessary?

Reply:
apparently not.

```
Undefined property: Illuminate\Database\Eloquent\Relations\HasMany::$map in /home/edho/osu/osu-web/app/Models/BeatmapDiscussion.php:156
```

┐(･_･)┌